docker/README.md

```diff
@@ -0,0 +1,6 @@
+## DeepSparse docker image
+This directory contains the Dockerfile to create minimal DeepSparse docker image.
+This image is based off the latest official Ubuntu image.
+
+In order to build and launch this image, run from the root directory:
+`docker build -t deepsparse_docker . && docker run -it deepsparse_docker`
```

Comment:
let's also give an example of invoking a script directly
`docker run deepsparse-image deepsparse.server ...`